lib/fs.js

```diff
@@ -1617,8 +1617,15 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (options === undefined)
```

Comment:
Because `fs.createReadStream()` currently allows any falsey value (`false`, `NaN`, etc.). Under this PR, passing in `false`, for example, would throw.